### PR TITLE
Pdf creator extension

### PR DESCRIPTION
**Pull request by pblanz**
fixed the issues with the overlapping tables (#258) and the one with the wrong table formatting (#111)

### CODE REVIEW

File: wp-content/plugins/pdf-creator-lite/functions.php [hunk 1]
```diff
@@ -8,7 +8,7 @@ function SSAPDF_addAdminPage ()
 {
 	$page_title = 'PDF Creator';
 	$menu_title = 'PDF Creator';
-	$capability = 'edit_pages';
+	$capability = 'manage_options';
```
Comment (Meash) [inline wp-content/plugins/pdf-creator-lite/functions.php]: What's this change about? It looks like only super administrators can create PDFs with this capability change

File: wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php [hunk 2]
```diff
@@ -106,7 +125,7 @@ function SSAPDFadminBuildPDF ()
 		switch_to_blog( $blogID );
 	}
 	
-	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are allowed to
+	if ( current_user_can( 'create_and_download_pdf' ) ) //Only let them download the file if they are admin
```
Comment (Meash) [inline wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php]: Regarding the comment: anyone with the `create_and_download_pdf` can do so right? That doesn't necessarily have to be an admin

File: wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php [hunk 3]
```diff
@@ -208,7 +226,13 @@ function ( $matches ) {
 					
 					$pdf->AddPage();				
 					$pdf->Bookmark( $title, $depth, 0, '', 'B', array($link_rgb['red'], $link_rgb['green'], $link_rgb['blue']), 0, '#TOC' );
-					
+
+					//XXX resolve bug with <p> tags within a table
```
Comment (Meash) [inline wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php]: has this bug been resolved?

Comment (pblanz) [inline wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php]: yes, i think so, but I only did minor tests. All tables from the Augsburg site looked fine.

File: wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php (hunk 2 above)
Comment (pblanz) [inline wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php]: I don't even remember doing those changes. Maybe this is due to a software update. I've recently downloaded the plugin and modified it to satisfy our needs. But I guess everyone can do so. I found those lines:
<pre>
add_action( 'wp_ajax_frontEndDownloadPDF', 'frontEndDownloadPDF' );			// fires if logged-in user
add_action( 'wp_ajax_nopriv_frontEndDownloadPDF', 'frontEndDownloadPDF' );	// fires if non-logged-in user
</pre>

File: wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php (hunk 3 above)
Comment (Meash) [inline wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php]: Okay let's remove the code-comment if nobody complains for a week ;)

**Pull request merged**